examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -64,6 +67,13 @@ public void testE2EWordCount() throws Exception {
     String e2eTestInputPath = "gs://apache-beam-samples/apache/LICENSE";
     options.setInputFile(e2eTestInputPath);
 
-    WordCount.main(TestPipeline.convertToArgs(options));
+    try {
+      WordCount.main(TestPipeline.convertToArgs(options));
+    } catch (Exception e) {
+      for (StackTraceElement elem : e.getStackTrace()) {
```

Comment:
This is equivalent to LOG.error("Failed to process WordCountIT:", e);

See: http://slf4j.org/faq.html#paramException